### PR TITLE
NoDrink component

### PR DESCRIPTION
**Pull request by DenisShvalov**
## About the PR
Added NoDrinkComponent and few change in DrinkSystem
Added NoDrink to Skeleton

## Why / Balance
If you want to make someone drinkless
+ Skeleton NoDrink anymore
## Media

https://github.com/space-wizards/space-station-14/assets/115770678/a8c4b041-b218-4d39-a060-e7f670a972de

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by UbaserB** [comment]
remove the CL, this isnt a playerfacing change

**Comment by DenisShvalov** [comment]
> remove the CL, this isnt a playerfacing change

Removed

**Comment by Anzarot121** [comment]
This should be added to the skeleton. At the moment, they cannot drink, but their attempts still remove substances from the vessels.

**Comment by Cyber-Finn** [comment]
silly question, but skeles drink milk to heal, right? Wouldnt this impact their ability to heal - if applied to them as-is (that is, removing the ability to drink anything at all)?

**Comment by DenisShvalov** [comment]
> silly question, but skeles drink milk to heal, right? Wouldnt this impact their ability to heal - if applied to them as-is (that is, removing the ability to drink anything at all)?

Skeleton can't drink milk, he heals by spilling milk on them

**Comment by Cyber-Finn** [comment]
> > silly question, but skeles drink milk to heal, right? Wouldnt this impact their ability to heal - if applied to them as-is (that is, removing the ability to drink anything at all)?
> 
> Skeleton can't drink milk, he heals by spilling milk on them

copy, thanks for answering! 😄

File: Content.Server/Nutrition/Components/NoDrinkComponent.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+using Content.Server.Nutrition.EntitySystems;
+
+namespace Content.Server.Nutrition.Components;
+
+/// <summary>
+/// This component marks entity, that can't drink anything
```
Comment (AJCM-git) [inline Content.Server/Nutrition/Components/NoDrinkComponent.cs:6]: Why cant you just remove the drink component from the entities that need this?

**Review by AJCM-git — CHANGES_REQUESTED** [review]

**Pull request closed without merging**